spacy/language.py

```diff
@@ -249,6 +252,16 @@ def config(self) -> Config:
     def config(self, value: Config) -> None:
         self._config = value
 
+    @property
+    def disabled(self) -> List[str]:
```

Comment:
The question is whether we should return a set here. I think it's a tough trade-off. How often will people just want to make an `in` check?

I think I lean towards returning a `FrozenSet`. Otherwise, a tuple is definitely better than a list, because a list allows `nlp.disabled.append`, which will always be wrong.

Reply:
I previously used a set here, but it made the user-facing property inconsistent with all the other similar properties and it was already a pain to document. Under the hood, `self._disabled` is a set.

Reply:
I don't think it's so inconsistent. Most of the rest of the things are lists because they are ordered, but why would people need to iterate over the list of names of disabled components in order? What you'll want to do with the disabled list is check against it. So I think a frozen set is a bit better.

Reply:
I don't think it's too bad to have the disabled components be returned in the same order they appear in the original components? Everything else would be kinda confusing.